client/common/directives/ModelFieldImage/ModelFieldImage.js

```diff
@@ -379,6 +379,11 @@ angular.module('dashboard.directives.ModelFieldImage', [
           $(window).off("mouseleave");
         });
 
+        scope.$on("removeModelFieldImage", function(event, key) {
+          if (key !== scope.key) return;
+          scope.clear(null, true);
```

Comment:
Is there a reason the negative case is used in the if statement instead of the positive? It could be one line e.g. `if (key === scope.key) scope.clear(null, true);`

Reply:
Not specific reason, I think I see it as a negative and then return out more often within the codebase so just went with that.